src/libraries/pkg/baseline/packageIndex.json

```diff
@@ -10,6 +10,13 @@
         "net45": "4.0.0.0"
       }
     },
+    "dotnet-ilverify": {
```

Comment:
I wouldn't expect any *additions* to the index as a result of this work.  You aren't adding new packages to the repo.  You should undo any new entries your process is creating.